README.md

```diff
@@ -17,20 +257,16 @@ year="2022",
 publisher="Springer International Publishing",
 address="Cham",
 pages="305--316",
-abstract="At the beginning of this year one of the authors read ``A brave new world'', a novel by Aldous Huxley. This book describes a dystopia, which anticipates the development of world-scale breeding technology, and how this technology creates the optimal human race. Taking into account that when talking about genetic algorithms our goal is to achieve the optimum solution of a problem, and this book kind of describes the process for making the ``perfect human'', or rather the ``perfect human population'', we will try to work on this parallelism in this paper, trying to find what is the key to the evolution processes described in the book. The goal is to develop a genetic algorithm based on the fecundation process of the book and compare it to other algorithms to see how it behaves, by investigating how the division in castes affects the diversity in the poblation. In this paper we describe the implementation of such algorithm in the programming language Julia, and how design and implementation decisions impact algorithmic and runtime performance.",
```

Comment:
It might be a bit verbose to add this abstract, but this change is outside the scope for this issue.